src/extras.jl

```diff
@@ -69,8 +71,8 @@ also accept them.
   the intervals; or a function `f(from, to, i; leftclosed, rightclosed)` that generates
   the labels from the left and right interval boundaries and the group index. Defaults to
   `"[from, to)"` (or `"[from, to]"` for the rightmost interval if `extend == true`).
-* `allowmissing::Bool=true`: when `true`, values outside of breaks result in missing values.
-  only supported when `x` accepts missing values.
+* `allowoutside::Bool=false`: when `true`, values outside of breaks result in missing values.
+  Only supported when `x` accepts missing values.
```

Comment:
if we change it and the default is `false` maybe also if it is `true` do promotion of `eltype` of `x` to ensure it allows missing values?

Reply:
Yeah, I guess that doesn't hurt. In that case the result will be inferred as a `Union`, but it's not worse than throwing an error. Luckily, the compiler is smart enough to infer the return type when the argument isn't passed and when the input already allows for missing values.